packages/client/hmi-client/src/components/workflow/ops/model-comparison/tera-model-comparison.vue

```diff
@@ -248,6 +262,17 @@ function runCode() {
 	emptyImages();
 	updateCodeState();
 
+	kernelManager.sendMessage('get_comparison_pairs_request', {}).register('any_get_comparison_pairs_reply', (data) => {
+		const comparisonPairs = data.msg.content?.return?.comparison_pairs;
+		const state = cloneDeep(props.node.state);
```

Comment:
No need to clone state here

Reply:
It's for the `else if`  when comparisonPairs returns nothing. I only update the titles using the past state if something has been stored previously